lib/components/viewers/stop-time-cell.tsx

```diff
@@ -19,6 +21,10 @@ const { getUserTimezone } = coreUtils.time
 const ONE_HOUR_IN_SECONDS = 3600
 const ONE_DAY_IN_SECONDS = 86400
 
+const PulsingRss = styled(Rss)`
+  animation: pulse-opacity 2s ease-in-out infinite;
+`
```

Comment:
Following up on https://github.com/opentripplanner/otp-react-redux/pull/661/files#r1018446380, insert the following and remove the corresponding inline style from the containing span (that style attribute actually has no effect, so maybe a transform is not needed at all):

```css
          transform: scaleX(-1);
```